src/styles/StyleUtils.js

```diff
@@ -45,17 +46,79 @@ function getAvatarStyle(size) {
     };
 }
 
+/**
+* Return the border style if avatar is SVG
+*
+* @param {String} size
+* @param {String} type
+* @returns {Object}
+*/
+function getAvatarBorderRadius(size, type) {
+    const AVATAR_BORDERS = {
+        [CONST.AVATAR_SIZE.SMALL_SUBSCRIPT]: variables.componentBorderRadiusSmall,
+        [CONST.AVATAR_SIZE.MID_SUBSCRIPT]: variables.componentBorderRadiusSmall,
+        [CONST.AVATAR_SIZE.SUBSCRIPT]: variables.componentBorderRadiusSmall,
+        [CONST.AVATAR_SIZE.SMALLER]: variables.componentBorderRadiusMedium,
+        [CONST.AVATAR_SIZE.SMALL]: variables.componentBorderRadiusMedium,
+        [CONST.AVATAR_SIZE.DEFAULT]: variables.componentBorderRadiusNormal,
+        [CONST.AVATAR_SIZE.MEDIUM]: variables.componentBorderRadiusLarge,
+        [CONST.AVATAR_SIZE.LARGE]: variables.componentBorderRadiusLarge,
+        [CONST.AVATAR_SIZE.LARGE_BORDERED]: variables.componentBorderRadiusRounded,
+    };
+
+    let borderRadius = variables.buttonBorderRadius;
+    if (type === CONST.ICON_TYPE_WORKSPACE) {
+        borderRadius = AVATAR_BORDERS[size];
+        return {borderRadius};
+    }
+    return {borderRadius};
+}
+
 /**
  * Return the border style if avatar is SVG
  *
- * @param {Boolean} isSVG
+ * @param {String} size
+ * @param {String} type
  * @returns {Object}
  */
-function getAvatarSVGBorder(isSVG) {
-    if (!isSVG) {
-        return {};
-    }
-    return styles.svgAvatarBorder;
+function getAvatarBorderStyle(size, type) {
+    return {
+        overflow: 'hidden',
+        ...getAvatarBorderRadius(size, type),
+    };
+}
+
+/**
+ * Helper method to return old dot default avatar associated with login
+ *
+ * @param {String} [workspaceName]
+ * @returns {Object}
+ */
+function getDefaultWorspaceAvatarColor(workspaceName) {
+    const colorOptions = [
+        ['#8DC8FF', '#003C73'],
+        ['#0185FF', '#002140'],
+        ['#003C73', '#8DC8FF'],
+        ['#8EECC4', '#085239'],
+        ['#03D47C', '#002E22'],
+        ['#085239', '#8EECC4'],
+        ['#FFED8F', '#722B03'],
+        ['#FED607', '#401102'],
+        ['#722B03', '#FFED8F'],
+        ['#FFC68C', '#780505'],
+        ['#FF7101', '#400000'],
+        ['#780505', '#FF7101'],
+        ['#FBCCFF', '#712A76'],
+        ['#F68DFE', '#49225B'],
+        ['#712A76', '#FBCCFF'],
+        ['#CCF7FF', '#28736D'],
+        ['#50EEF6', '#134038'],
+        ['#28736D', '#CCF7FF'],
```

Comment:
@shawnborton do we have names for all these brand colors somewhere in the figma?

Reply:
NVM, Found it!

Reply:
Would you rather I add only the colors that are used, or import all the brand colors (100-800 colors) into `colors.js` and potentially their own file?

Reply:
Example of a color clean up here: https://github.com/Expensify/App/compare/georgia-brand-colors?expand=1

Reply:
My opinion, we should focus on the colors that are used? and have a follow up PR for the clean up?

Reply:
Yes, I meant on a follow up PR, I should have specified.

Reply:
I like the idea of having support for all of our colors in the app, and it can live in a different file. 

It would be nice to be able to easily access colors by saying something like give me `green.100` or something.